content/docs/user-guide/visualizing-plots.md

```diff
@@ -0,0 +1,385 @@
+# Visualizing Plots
+
+A typical workflow for DVC plots is:
+
+1. Save data to a [supported file format](#supported-file-formats).
+
+```csv
+fpr,tpr,threshold
+0.0,0.0,1.5
+1.0,1.0,0.0
+```
+
+2. [Define plots](#defining-plots), optionally using
+   [templates](#plot-templates-data-series-only) to configure how to visualize
+   the data.
+
+```yaml
+plots:
+  auc.tsv:
+    x: fpr
+    y: tpr
+```
+
+3. [Show](/doc/command-reference/plots/show) all plots in a single view or
+   report.
+
+![](/img/plots_prc_get_started_show.svg)
+![](/img/plots_roc_get_started_show.svg)
+![](/img/plots_importance_get_started_show.png '=300 :wrap-left')
+![](/img/plots_cm_get_started_show.svg)
+
+4. Run [experiments](/doc/user-guide/experiment-management/experiments-overview)
+   and [compare](/doc/command-reference/plots/diff) plots.
+
+![](/img/plots_prc_get_started_diff.svg)
+![](/img/plots_roc_get_started_diff.svg)
+![](/img/plots_importance_get_started_diff.png)
+
+## Generating plots files
+
+To generate the data files for plots, you can:
+
+1. Use [DVCLive](/doc/dvclive/dvclive-with-dvc) in your Python code to log the
+   data in the expected format for you.
+2. Save data yourself in one of the
```

Comment:
💅🏼 Maybe a regular bullet list (not numbered) would be better here to signal equal alternatives and contrast to previous numbered list which was a list of steps.